### PR TITLE
container: add ConsoleLog() API extension

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: container.go [hunk 1]
```diff
@@ -1866,3 +1867,43 @@ func (c *Container) SetRunningConfigItem(key string, value string) error {
...
 	return nil
 }
+
+// ConsoleLog allows to perform operations on the container's in-memory console
+// buffer.
+func (c *Container) ConsoleLog(opt ConsoleLogOptions) ([]byte, error) {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+
+	lxcConsoleLog := C.struct_lxc_console_log{
```
Comment (caglar10ur) [inline container.go]: I know I'm just nitpicking but lxcConsoleLog is a long variable name :) How about cl?

**Review by caglar10ur — APPROVED** [review]

File: container.go (hunk 1 above)
Comment (brauner) [reply container.go]: Sure, I'll update. :)

Comment (brauner) [reply container.go]: @caglar10ur updated.

**Comment by caglar10ur** [comment]
LGTM! Thanks @brauner

**Pull request merged**